code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -28,6 +29,13 @@
 
 
 /obj/item/weapon/spellbook/attack_self(mob/user as mob)
+	if(!owner)
+		user << "You bind the spellbook to yourself."
+		owner = user
+		return
+	if(user != owner)
+		user << "You can't open the spellbook somehow."

```

Comment:
How about something more nerdy? Such as "a mysterious force prevents you opening the [name]" or something.